src/components/media/MediaSettingsModal.jsx

```diff
@@ -94,10 +94,10 @@ export default class MediaSettingsModal extends Component {
           <Toast notificationType='error' text={`Another ${type === 'image' ? 'image' : 'file'} with the same name exists. Please choose a different name.`}/>, 
           {className: `${elementStyles.toastError} ${elementStyles.toastLong}`}
         );
-      } else if (err?.response?.status === 413) {
-        // Error due to file size too large
+      } else if (err?.response?.status === 413 || err?.response === undefined) {
+        // Error due to file size too large - we receive 413 if nginx accepts the payload but it is blocked by our express settings, and undefined if it is blocked by nginx
         toast(
-          <Toast notificationType='error' text={`The ${type === 'image' ? 'image' : 'file'} is too large. Please choose a smaller ${type === 'image' ? 'image' : 'file'}.`}/>, 
+          <Toast notificationType='error' text={`Unable to upload as ${type === 'image' ? 'image' : 'file'} size is too large. Please reduce your ${type === 'image' ? 'image' : 'file'} size and try again.`}/>, 
```

Comment:
"as the file size exceeds 5mb" instead of "size is too large"

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/299/commits/684120b4c5838f026d524047aa49a05d80aa5645